types/model.d.ts

```diff
@@ -243,6 +244,85 @@ export declare class ModelBase {
     anotherClass: string | ModelBaseClass | object,
     options?: Options,
   ): ModelBaseClass;
+
```

Comment:
why just the `model.d.ts` file, why not `model.ts` ?

Reply:
I am afraid I don't understand the question. I am editing an existing TypeScript declaration file which is providing type information for artifacts implemented by JavaScript code in `lib`. There is no `model.ts` file in juggler.

See "What is a Declaration File?" in https://medium.com/razroo/what-are-declaration-files-in-typescript-476c1c06f353#936b for a decent introduction into type declarations, and [Introduction to Declaration Files](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html) in TypeScript HandBook for content going more deep.

Reply:
@bajtos , I haven't worked with `.d.ts` files yet. I know that they are one of the file types generated by a transpilation of Typescript to Javascript. I will read the article you mentioned.